libs/tokens/src/updaters/TokensListsUpdater/index.ts

```diff
@@ -67,5 +67,20 @@ export function TokensListsUpdater({ chainId: currentChainId }: TokensListsUpdat
     upsertLists(chainId, listsStates)
   }, [listsStates, isLoading, chainId, upsertLists, setTokenListsUpdating, updateLastUpdateTime])
 
+  // Check if a user is from US and use Uniswap list, because of the SEC regulations
+  useEffect(() => {
+    fetch('https://api.country.is')
+      .then((res) => res.json())
+      .then(({ country }) => {
+        const isUsUser = country === 'US'
+
+        if (isUsUser) {
+          setEnvironment({ useUniswapListOnly: true })
+          updateLastUpdateTime({ [chainId]: 0 })
+        }
+      })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```

Comment:
The check is only ran once on load and never updated unless there's a refresh?

Reply:
Yes, it works like that.
I don't think we should poll it, in 99.99% cases people don't change their location dynamically.
I hope people who are switching VPN know they should refresh page :)

Reply:
> I hope people who are switching VPN know they should refresh page :)

ahaha, agree